custom_nodes/.changeset/chilled-rings-brake.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
What is this file?

Reply:
This file is a changeset that records what this PR changed, what packages are affected, and how those packages' versions should be bumped (major / minor / patch). When we publish, I run `changeset version`, and it automatically updates all the package.json files with the correct version increase. Then, it deletes all these pending changeset files and puts them in the changelog for each package.

Reply:
nice!!